op-service/log/cli.go

```diff
@@ -230,7 +232,10 @@ func NewLogger(wr io.Writer, cfg CLIConfig) log.Logger {
 // Geth and other components may use the global logger however,
 // and it is thus recommended to set the global log handler to catch these logs.
 func SetGlobalLogHandler(h slog.Handler) {
-	log.SetDefault(log.NewLogger(h))
+	l := log.NewLogger(h)
+	ctx := logfilter.AddLogAttrToContext(context.Background(), "global", "true")
```

Comment:
```suggestion
	ctx := logfilter.AddLogAttrToContext(context.Background(), "global", true)
```
looks like bools are handled by slog